docs/content/Coding Milestones/PoC2023q1/example1.md

```diff
@@ -446,7 +446,7 @@ spec:
         ports:
         - name: http
           containerPort: 80
-          hostPort: 8081
```

Comment:
In addition to the coincidence on port number, there is a coincidence on field name.  In a kind Cluster object, `hostPort` refers to a port on the host of the kind cluster, not the Kube node inside the cluster.  OTOH, in a Kubernetes Pod spec template, `hostPort` refers to a port on the Kube node hosting that pod.

In short, I think you do not want to make a change here (since you are not changing the `containerPort` in the kind Cluster).

Reply:
fixed